sql/updates/world/2015_08_29_99_world.sql

```diff
@@ -0,0 +1,268 @@
+SET @OGUID := 345000; -- 20 required
+SET @CGUID := 600000;
+
+SET @NPC := @CGUID+1;
+SET @PATH := @NPC * 10;
+
+UPDATE `creature_template` SET `difficulty_entry_1`=0 WHERE `entry` IN (25740,25755,25756,25865);
+UPDATE `creature_template` SET  `minlevel`=82,`maxlevel`=82, `ScriptName`='boss_ahune' WHERE `entry`=25740; -- Ahune
+UPDATE `creature_template` SET  `minlevel`=80,`maxlevel`=80, `AIName`='SmartAI' WHERE `entry`=25755; -- Ahunite Hailstone
+UPDATE `creature_template` SET  `minlevel`=80,`maxlevel`=80, `AIName`='SmartAI' WHERE `entry`=25756; -- Ahunite Coldwave
+UPDATE `creature_template` SET  `minlevel`=80,`maxlevel`=80, `AIName`='SmartAI' WHERE `entry`=25757; -- Ahunite Frostwind
+UPDATE `creature_template` SET  `minlevel`=80,`maxlevel`=80, `ScriptName`='npc_frozen_core' WHERE `entry`=25865; -- Frozen Core
+UPDATE `creature_template` SET `ScriptName`='npc_ahune_bunny' WHERE `entry`=25745;
+UPDATE `creature_template` SET `unit_flags`=33554432, `MovementType`=2 WHERE `entry` IN (25964,25965,25966); -- Shaman beam bunny
+UPDATE `creature_template` SET `unit_flags`=33554432 WHERE `entry`=26239; -- Ghost of Ahune
+UPDATE `creature_template` SET `flags_extra`=128 WHERE `entry`=25985; -- Ahune Ice Spear Bunny
+UPDATE `gameobject_template` SET `ScriptName`='go_ahune_ice_stone' WHERE `entry`=187882;
+UPDATE `creature` SET `orientation`=2.408554 WHERE `guid`=202734; -- Luma
+UPDATE `creature` SET `orientation`=3.804818 WHERE `guid`=202737; -- Flamecaller
+UPDATE `creature_template` SET `HealthModifier`='94.5' WHERE `entry`=25865;
+UPDATE `creature_template` SET `HealthModifier`='18.8' WHERE `entry`=25755;
+UPDATE `creature_template` SET `HealthModifier`='3.538' WHERE `entry`=25756;
+UPDATE `creature_template` SET `HealthModifier`='1.5' WHERE `entry`=25757;
+UPDATE `creature_template` SET `HealthModifier`=4 WHERE `entry`=40446;
+UPDATE `creature_template` SET `InhabitType`=4 WHERE `entry` IN (25964,25965,25966,26190);
+
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (46430,46371,45926,46878,46320,46146,46398,46843);
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
```

Comment:
`(46398, 'spell_ice_bombardment_dest_picker')` is not added after the `DELETE FROM` command has removed 46398. Is this intentional?